third_party/java/arcs/sdk/kotlin/BUILD

```diff
@@ -4,5 +4,10 @@ licenses(["notice"])
 
 alias(
     name = "kotlin",
-    actual = "//src/wasm/kotlin:arcs_wasm",
+    actual = "//src/wasm/kotlin:arcs",
```

Comment:
Again, I'm curious about these aliases. Can you document why they're needed? Maybe add TODO?

Reply:
I think these are needed to have simultaneous JVM and Wasm builds, while referring to a single build target `...:kotlin`.

Reply:
I'll leave a quick comment.

Reply:
This seems kind of strange to me, since our other targets aren't named like that. I'm not really sure anyone's workflow does that tbh. I'd suggest just removing these.